cmd/file_patch.go

```diff
@@ -112,52 +112,55 @@ func newPatchCmd() *cobra.Command {
+The input file is read, the patches are applied, and if successful, written
 to the output file. The patches can be specified by a '--selector' and one or more
-'--value' tags, or via patch-files.
-
-When using '--selector' and '--values', the items will be selected by the 'selector' which is
-a JSONpath query. From the array of nodes found, only the objects will be updated.
-The 'values' will be applied on each of the JSONobjects returned by the 'selector'.
-
-The value part must be a valid JSON snippet, so make sure to use single/double quotes
-appropriately. If the value is empty, the field will be removed from the object.
-Examples:
-  --selector="$..services[*]" --value="read_timeout:10000"
-  --selector="$..services[*]" --value='_comment:"comment injected by patching"'
-  --selector="$..services[*]" --value='_ignore:["ignore1","ignore2"]'
-  --selector="$..services[*]" --value='_ignore:' --value='_comment:'
-
-The patch-files have the following format (JSON or Yaml) and can contain multiple
-patches that will be applied in order;
-
-  { "_format_version": "1.0",
-    "patches": [
-      { "selectors": [
-					"$..services[*]"
-				],
-        "values": {
-          "read_timeout": 10000,
-          "_comment": "comment injected by patching"
-        },
-        "remove": [ "_ignore" ]
-      }
-    ]
-  }
+'--value' tags, or via patch files.
+		
+When using '--selector' and '--values', the items are selected by the 'selector', 
+which is a JSONpath query. From the array of nodes found, only the objects are updated.
+The 'values' are applied on each of the JSONObjects returned by the 'selector'.
+
+The value must be a valid JSON snippet, so use single/double quotes
+appropriately. If the value is empty, the field is removed from the object.
+
+Examples of valid values:
+
+	--selector="$..services[*]" --value="read_timeout:10000"
+	--selector="$..services[*]" --value='_comment:"comment injected by patching"'
+	--selector="$..services[*]" --value='_ignore:["ignore1","ignore2"]'
+	--selector="$..services[*]" --value='_ignore:' --value='_comment:'
+
+
+Patch files have the following format (JSON or YAML) and can contain multiple
+patches that are applied in order:
+
+	{ "_format_version": "1.0",
+		"patches": [
+		{ "selectors": [
+			"$..services[*]"
+			],
+			"values": {
+			"read_timeout": 10000,
+			"_comment": "comment injected by patching"
+			},
+			"remove": [ "_ignore" ]
+		}
+		]
+	}
 `,
 		RunE: executePatch,
 	}
 
 	patchCmd.Flags().StringVarP(&cmdPatchInputFilename, "state", "s", "-",
-		"decK file to process. Use - to read from stdin")
+		"decK file to process. Use - to read from stdin.")
 	patchCmd.Flags().StringVarP(&cmdPatchOutputFilename, "output-file", "o", "-",
-		"output file to write. Use - to write to stdout")
+		"Output file to write. Use - to write to stdout.")
 	patchCmd.Flags().StringVarP(&cmdPatchOutputFormat, "format", "", "yaml",
-		"output format: yaml or json")
+		"Output format: yaml or json.")
```

Comment:
This command accepts lowercase `yaml` or `json`, whereas all the other `deck file` commands expect uppercase, eg https://github.com/Kong/deck/pull/968/files#diff-941212cced7cf3d08905ca571d87840a3b28f041a83619bd0fd681f864de6861R106. Was this intentional? Any chance this could be made consistent to avoid confusion for users?

Reply:
technically all commands accept any casing (they are case insensitive)

Reply:
https://github.com/Kong/deck/pull/970 will change them to lowercase

Reply:
(with that I mean the help text is now lowercase, input validation remains case-insensitive)